_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -41,6 +46,10 @@ function __construct() {
 		self::$block_page_rendering_for_idc = (
 			Jetpack::validate_sync_error_idc_option() && ! Jetpack_Options::get_option( 'safe_mode_confirmed' )
 		);
+		if( $_GET[ 'page' ] == 'jetpack' ) {
+			add_filter( 'nocache_headers', 'add_no_store_header' );
+			nocache_headers();
```

Comment:
Is it necessary to call `nocache_headers()` again? It will be called early in `wp-admin/admin.php` anyway, so why call it again?

Reply:
because I'm paranoid and I don't trust that we are not running some customized install that actually doesn't add the cache headers by default... AKA "just in case"

(you are right, it's probably not needed... but it also doesn't do any harm to be sure :D )

Reply:
Oh well, if anyone modifies this in the WP core, I'd say they should be responsible for it. Other weird things would happen, and sky would fall, especially if disabled for admin-ajax.php for example.